src/Auth0.AuthenticationApi/Builders/AuthorizationUrlBuilder.cs

```diff
@@ -13,13 +13,6 @@ namespace Auth0.AuthenticationApi.Builders
     /// </remarks>
     public class AuthorizationUrlBuilder : UrlBuilderBase<AuthorizationUrlBuilder>
     {
-        private static readonly Dictionary<AuthorizationResponseType, string> Map = new Dictionary<AuthorizationResponseType, string>
-        {
-            { AuthorizationResponseType.Code, "code" },
-            { AuthorizationResponseType.IdToken, "id_token" },
-            { AuthorizationResponseType.Token, "token" }
-        };
```

Comment:
Replaced this map with extension methods so it can be used outside of this UrlBuilder